src/main/java/study/racinggame/domain/Car.java

```diff
@@ -1,30 +1,32 @@
 package study.racinggame.domain;
 
+import study.racinggame.domain.strategy.MoveStrategy;
+
 public class Car {
-    private int position;
-    private Name name;
+    private final Position position;
+    private final Name name;
 
     public Car(String name) {
         this.name = new Name(name);
+        this.position = new Position();
+    }
+
+    public Car(int position, String name) {
+        this.position = new Position(position);
+        this.name = new Name(name);
     }
```

Comment:
```suggestion
    public Car(Position position, Name name) {
        this.position = position;
        this.name = name;
    }
```
주생성자(특별한 변환 작업 없이 인스턴스 변수에 값을 할당)를 생성한 후 this(new Position(), new Name(name));과 같이 주 생성자를 호출하는 구조로 구현하는 습관을 들이는 것은 어떨까?

인자의 타입이 다른 다수의 생성자를 추가하는 것은 권장하는 개발 방식이다.